### DISCUSSION TITLE
我好像猜到黑榜的算法了

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#635] [General]
已知条件：

1. 现在返回的信息当中有楼层信息， https://github.com/Arondight/Adachi-BOT/issues/515#issuecomment-1029836431 ；
2. 如果角色出战次数一样（例如都是12次），那么每次随机显示；

所以理论上只要查询次数足够多，就可以获得全部出战的角色

**Comment by Arondight** [comment]
满足这个条件的太少了，大多数人打深渊，12 层的永远都显示不全
